### DISCUSSION TITLE
Need a Tech Doc filled out

### DISCUSSION THREAD

**Discussion by thayes2714** [ivadomed/ivadomed#1078] [Q&A]
Hi there!  I'm with the University of Texas Dental School and we've had a request to have AxonDeepSeg installed on a computer at our institution.  It has to go through a security review first before we can install it and I need to have a questionnaire filled out for it.  Can I get an email for the contact person who would be able to help me with this?

Thanks
Traci Hayes

**Comment by jcohenadad** [comment]
Hi, AxonDeepSeg is an open-source package written in Python. I believe your university does not require to fill up a security document for every python library installed (numpy, scipy, etc.). Could you please elaborate on the need for filling such a document? We are a relatively small core development team so I'm sure you will understand that we have limited capacity to support such requests.

**Comment by thayes2714** [comment]
[computer_device_questionnaire.docx](https://github.com/ivadomed/ivadomed/files/8063597/computer_device_questionnaire.docx)


Thank you for responding.  This is the questionnaire we ask be filled out so it can be determined if the software/program will work with our network and policies as well as guide us into determining if it needs to go through an IT Security Review.  This is a general document that is used for all of our different types of acquisitions so not all the questions would pertain to this.  We're mostly concerned with how/if the program would connect to our network, how information is loaded/stored into the program, and if any PHI is shared/stored/used within the program.

**Reply by jcohenadad** [reply]
The program is standalone and only connects to the internet during installation. There is no private information sent. @dyt811 maybe we should consider adding a page in the documentation that includes all these disclaimers?

**Reply by thayes2714** [reply]
Can you tell me if any patient data is loaded into or used within the program at all?  If so, can you confirm it will not be saved in the program?  I'm needing this for a security review.

**Reply by jcohenadad** [reply]
could you please elaborate what you mean by "patient data"? The image (in NIfTI format) is loaded because it is required to do any sort of analysis on them
